whisper_online.py

```diff
@@ -430,10 +430,18 @@ def split(self, text):
                 return tokenize_uk.tokenize_sents(text)
         return UkrainianTokenizer()
 
-    # supported by mosestokenizer (1.2.1)
     if lan in "as bn ca cs de el en es et fi fr ga gu hi hu is it kn lt lv ml mni mr nl or pa pl pt ro ru sk sl sv ta te yue zh".split():
-        from mosestokenizer import MosesSentenceSplitter
-        return MosesSentenceSplitter(lan)
+        class Moses_Tokenizer:
+            def split(self, text):
+                try:
+                    # supported by fast-mosestokenizer
+                    from mosestokenizer import MosesTokenizer
+                    return MosesTokenizer(lan).split(text)
+                except AttributeError: ### IF APPLE SILICON ### 
+                    # supported by mosestokenizer (1.2.1)
+                    from mosestokenizer import MosesSentenceSplitter
+                    return MosesSentenceSplitter(lan)(text)
+        return Moses_Tokenizer() 
```

Comment:
thanks, it's better, but please:
1) "try: import MosesTokenizer" outside the class. 
2) And then "return MosesTokenizer(lan)" . Don't wrap it in the Moses_Tokenizer object
3) Don't import in every split function call